tests/spread/integration/wget/task.yaml

```diff
@@ -0,0 +1,11 @@
+summary: Integration tests for wget
+
+execute: |
+  # Chisel a minimum number of slices to give us a runnable system that we can
+  # test in.
+  rootfs="$(install-slices ca-certificates_data wget_bins)"
+  cp /etc/resolv.conf "${rootfs}/etc/"
+
+  # Download README.md from repo and compare to a known hash.
+  chroot "${rootfs}/" wget https://raw.githubusercontent.com/canonical/chisel-releases/refs/heads/ubuntu-24.04/README.md
```

Comment:
I am afraid this test may break some time in the future when any change occur to the README.md file, which will change the file's checksum as a result.

I'd suggest instead to use a released binary with a known checksum to download. For instance, a chisel release binary seems like a perfect option:
https://github.com/canonical/chisel/releases/download/v1.0.0/chisel_v1.0.0_linux_amd64.tar.gz
https://github.com/canonical/chisel/releases/download/v1.0.0/chisel_v1.0.0_linux_amd64.tar.gz.sha384

Reply:
Not a problem, adjusted as per the request.